packages/oruga-next/src/components/input/Input.vue

```diff
@@ -365,6 +370,14 @@ function rightIconClick(event: Event): void {
 const isPasswordVisible = ref(false);
 const inputType = ref(props.type);
 
+const computedInputType = computed(() => {
+    if (props.passwordReveal) {
```

Comment:
Again, this seems to be better placed in it's own PR :)

Reply:
Ok I can break it out.